extensions/amp-story-auto-ads/0.1/story-ad-button-text-fitter.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '../../../src/services';
+import {px, setStyle, setStyles} from '../../../src/style';
+
+/** @enum {number} From design spec. */
+const FontSizes = {
+  MIN: 12,
+  MAX: 14,
+};
+
+export class ButtonTextFitter {
+  /**
+   * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
+   */
+  constructor(ampdoc) {
+    /** @private @const {!../../../src/service/resources-interface.ResourcesInterface} */
+    this.resources_ = Services.resourcesForDoc(ampdoc);
+
+    /** @private {!Document} */
+    this.doc_ = ampdoc.win.document;
+
+    /** @private {!Element} */
+    this.measurer_ = this.doc_.createElement('div');
+
+    /** @private {number} Fixed button height from design spec. */
+    this.maxHeight_ = 32;
```

Comment:
Is this always fixed? Should it defined as a constant?

Reply:
Moved to const